src/components/Layout/BurgerMenu/BurgerMenuPushableLayout.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+
+import useResponsive from 'decentraland-gatsby/dist/hooks/useResponsive'
+import Responsive from 'semantic-ui-react/dist/commonjs/addons/Responsive'
+
+import { useBurgerMenu } from '../../../hooks/useBurgerMenu'
+
+interface Props {
+  children: React.ReactNode
+}
+
+function BurgerMenuPushableLayout({ children }: Props) {
+  const burgerMenu = useBurgerMenu()
+  const responsive = useResponsive()
+  const isMobile = responsive({ maxWidth: Responsive.onlyMobile.maxWidth })
+
+  if (isMobile) {
+    return (
+      <div
+        className="Animated"
+        style={burgerMenu?.status.open ? { transform: `translateY(${burgerMenu.status.translate})` } : {}}
```

Comment:
I'd love to see this in a class BurgerMenu--open 😏

Reply:
We can't because translate is a variable that can take many values :)